shared/admin-tools/user/rotate-environment-secrets.ts

```diff
@@ -74,6 +74,7 @@ const rotateSecrets = async (environmentName: string): Promise<void> => {
 
   const USTC_ADMIN_PASS = makeNewPassword();
 
+  // Should only be run locally; is it necessary?
```

Comment:
Yes, this only run locally, and it's purpose is to rotate the passwords and make them available for consumption. This is how we distribute it to the non-technical users on the team. There's always a better way...

Reply:
Thank you!